src/test/codegen/unwind-extern.rs

```diff
@@ -0,0 +1,15 @@
+// compile-flags: -C opt-level=0
+
+#![crate_type = "lib"]
+#![feature(unwind_attributes)]
+
+// make sure these all do *not* get the attribute
+// CHECK-NOT: nounwind
+
+pub extern fn foo() {} // right now we don't abort-on-panic, so we also shouldn't have `nounwind`
+#[unwind(allowed)]
+pub extern fn foo_allowed() {}
```

Comment:
This should `CHECK-NOT: nounwind` on both nightly and stable.

Reply:
AFAIK the one `CHECK-NOT` above just checks that the string does not occur at all in the entire LLVM IR file? Is that not how it works?

Can you point me to documentation describing precisely how `CHECK-NOT` works?

Reply:
> FAIK the one CHECK-NOT above just checks that the string does not occur at all in the entire LLVM IR file? Is that not how it works?

Oh, I thought one had to use it once per function (the docs are here but I have to look them up every time: https://llvm.org/docs/CommandGuide/FileCheck.html)

Reply:
So do you agree that this test tests the right thing?

Reply:
Yes, this should not be `nounwind`.

Reply:
> but if you want to sanity-check it, try something that does exist and see if it fails. smile

Done, and it does.